### PR TITLE
fix(chat): align version selector for models with long description (Issue #1346)

### PR DESCRIPTION
**Pull request by likeamike**
**Description:**

Fix the alignment of the version selector for models with long descriptions

Issues:

- Issue #1346

**UI changes**

before fix:
image

after fix:
image

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1385.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Chat/ModelList.tsx [hunk 1]
```diff
@@ -111,6 +111,7 @@ const ModelGroup = ({
                 : currentEntity.name}
             </span>
             <ModelVersionSelect
+              className="absolute right-3 !h-auto"
```
Comment (denys-kolomiitsev) [inline apps/chat/src/components/Chat/ModelList.tsx]: It's not a good practice to use **!important**.
Could it be avoided if possible?

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1385.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by denys-kolomiitsev — APPROVED** [review]

File: apps/chat/src/components/Chat/ModelList.tsx (hunk 1 above)
Comment (likeamike) [reply apps/chat/src/components/Chat/ModelList.tsx]: done

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1385.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1385.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1385.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1385.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1385.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1385.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**